database/schema/12-swap.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE swap
+(
+    timestamp           TIMESTAMP WITHOUT TIME ZONE NOT NULL REFERENCES block (timestamp),
+    hash                TEXT    NOT NULL REFERENCES transaction (hash),
+    height              BIGINT  NOT NULL REFERENCES block (height),
+
+    /* Msgs */
+    routes              JSONB   NOT NULL DEFAULT '[]'::JSONB,
+    sender              VARCHAR(43) NOT NULL,
```

Comment:
don't know is it right or not, but maybe something NOT NULL REFERENCES account (address) ??
also maybe rename it to address?